Tests/Tests/JSONs/dynamic.json

```diff
@@ -127,9 +127,9 @@
             }
           },
           {
-            "id":"contacts[:index].phone_number",
-            "title":"Phone number",
-            "type":"number",
+            "id":"contacts[:index].last_name",
```

Comment:
I had to change this because it's better to not  have the same collection of fields everywhere. (testing)